frontend/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "license": "MIT",
   "scripts": {
+    "prettier-watch": "onchange \"**/*\" -- prettier --write --ignore-unknown {{changed}}",
```

Comment:
**angular/webpack** already watches for file changes, that's what enables that handy browser refresh whenever we save a file while developing. we need to make sure that this new **`onchange`** file watcher doesn't interfere with it -- having two processes that both watch for, and make, changes to the same files can cause problems. If they _do_ work in tandem and "play nice together" than this is perfect!

Reply:
So, after testing some more, everything looks to be playing nice from what I can tell. I had a bit of lag at first. But not for too long and that could be my machine. Do you wanna pull down this branch and try it out?